src/testing/account.ts

```diff
@@ -36,7 +37,11 @@ export async function getTestAccount(
   const algorand =
     algodOrAlgorandClient instanceof AlgorandClient
       ? algodOrAlgorandClient
-      : AlgorandClient.fromClients({ algod: algodOrAlgorandClient, kmd })
+      : AlgorandClient.fromClients({
+          algod: algodOrAlgorandClient,
+          kmd:
```

Comment:
`ClientManager.getConfigFromEnvironmentOrLocalNet` is probably better to avoid the duplication of KMD logic.

Reply:
I was thinking of using this, however previous code didn't have a LocalNet fallback, so it's behaviourally different. Given it should only be used for testing, it should be safe.

Reply:
Updated.